### DISCUSSION TITLE
Unable to send BNB from the Sovryn wallet

### DISCUSSION THREAD

**Discussion by EduardKostikov** [defray-labs/sovryn-mobile#146] [Q&A] [answered]
Hello All! 👋 
Has anyone tried to send BNB (on bsc chain) from the Sovryn wallet? It doesnt work. The funds sent on bsc network get stuck in the wallet. Is there a workaround? Thanks! 🙏

**Comment by grinry** [comment, marked as answer]
Hey,
Please look at this comment https://github.com/leverim/sovryn-mobile/issues/145#issuecomment-1973020643 on the issue reported.

Using non RSK have bug in the current app release and fixed release is not accepted by apple app store, so app will be deprecated.
You will have to export your wallet to MetaMask for example to send out BNB.
